### PR TITLE
aura: add page

### PR DESCRIPTION
**Pull request by AakashSharma7269**
- [🗸 ] The page (if new), does not already exist in the repository.
- [🗸 ] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [ 🗸] The page has 8 or fewer examples.
- [🗸 ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [🗸 ] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ 🗸] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by AakashSharma7269** [comment]
Aura has an extensive list of commands that makes it a very unique AUR helper and package manager, I tried to collect only the core essential commands which is often used by aura users. I do understand that 16 commands is a bit too much for TL;DR pages and I am willing to remove whatever is not needed as per instructions from the reviewers.

**Comment by navarroaxel** [comment]
Hi @AakashSharma7269, welcome and thanks for your submission. The pages can't have more than 8 example, but you can create a page for the subcommands, like  `pacman-sync` for `pacman`.

**Comment by AakashSharma7269** [comment]
Hey @navarroaxel I have a removed a few commands, I hope it's fine now.

File: pages/linux/aura.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# aura
+
+> The Aura Package Manager: A secure, multilingual package manager for Arch Linux and the AUR.
+> More information: <https://github.com/fosskers/aura>.
+
+- Search for packages from the repos and AUR:
```
Comment (marchersimon) [inline pages/linux/aura.md:6]: I can't find the `--both` flag on the GitHub page, but I assume it enables the official repos too, right?

File: pages/linux/aura.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- Update all system repo packages:
+
+`aura -Syu`
+
+- Downgrade a package from the package cache:
+
+`aura -C {{package_name}}`
+
+- Remove an installed package and both its dependencies:
```
Comment (marchersimon) [inline pages/linux/aura.md:30]:
```suggestion
- Remove both an installed package and its dependencies:
```

**Review by marchersimon — COMMENTED** [review]
Just two small suggestions.

File: pages/linux/aura.md (hunk 1 above)
Comment (AakashSharma7269) [reply pages/linux/aura.md:6]: Yes all the repos in /etc/pacman.conf
And it's in the man page.

File: pages/linux/aura.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# aura
+
+> The Aura Package Manager: A secure, multilingual package manager for Arch Linux and the AUR.
+> More information: <https://github.com/fosskers/aura>.
+
+- Search for packages from the repos and AUR:
+
+`aura -As --both {{package_name|search_regex}}`
```
Comment (navarroaxel) [inline pages/linux/aura.md:8]: What kind of regex this supports?

File: pages/linux/aura.md (hunk 1 above)
Comment (navarroaxel) [reply pages/linux/aura.md:6]:
```suggestion
- Search for packages from the official repositories and AUR:
```

File: pages/linux/aura.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+- Update all AUR packages in a verbose mode and remove all make dependencies:
+
+`aura -Akuax`
+
+- Install a package from the AUR:
+
+`aura -A {{package_name}}`
+
+- Install a package from the system repos:
```
Comment (navarroaxel) [inline pages/linux/aura.md:18]:
```suggestion
- Install a package from the official repositories:
```

File: pages/linux/aura.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+`aura -S {{package_name}}`
+
+- Update all system repo packages:
```
Comment (navarroaxel) [inline pages/linux/aura.md:22]:
```suggestion
- Synchronize and update all packages from the official repositories:
```

File: pages/linux/aura.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+`aura -Syu`
+
+- Downgrade a package from the package cache:
```
Comment (navarroaxel) [inline pages/linux/aura.md:26]: Is `using` better? :thinking: 
```suggestion
- Downgrade a package using the package cache:
```

File: pages/linux/aura.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+`aura -C {{package_name}}`
+
+- Remove both an installed package and its dependencies:
```
Comment (navarroaxel) [inline pages/linux/aura.md:30]:
```suggestion
- Remove a package and its dependencies:
```

File: pages/linux/aura.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+
+`aura -Rs {{package_name}}`
+
+- Uninstall all orphan packages:
```
Comment (navarroaxel) [inline pages/linux/aura.md:34]:
```suggestion
- Remove orphan packages (installed as dependencies but not required by any package):
```

**Review by navarroaxel — COMMENTED** [review]
We prefer long arguments if these are available (take a look to the `pacman` page). Could we use long ones for this page?

File: pages/linux/aura.md (hunk 6 above)
Comment (AakashSharma7269) [reply pages/linux/aura.md:26]: Yea that makes sense, i'll commit that.

**Comment by AakashSharma7269** [comment]
About the regex support, i'm not entirely sure about that and i will check in on the main repo and let you know. 
I found long arguments for most of the commands although it makes it look messy, i'll be committing the changes soon. (Note: i was unable to find a long argument for -S)

File: pages/linux/aura.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+
+`aura -Sy --sysupgrade`
+
+- Downgrade a package using the package cache:
+
+`aura --downgrade {{package_name}}`
+
+- Remove a package and its dependencies:
+
+`aura -Rsu {{package_name}}`
```
Comment (navarroaxel) [inline pages/linux/aura.md:32]:
```suggestion
`aura --remove --recursive --unneeded {{package_name}}`
```

**Review by navarroaxel — APPROVED** [review]

**Review by marchersimon — APPROVED** [review]
Thanks @AakashSharma7269

**Comment by AakashSharma7269** [comment]
Glad I could help.

**Pull request merged**